### PR TITLE
Support auto-sizing without react-sizeme

### PR DESCRIPTION
**Pull request by evancharlton**
Break the dependency on react-sizeme, and size the chart to be the size of its container.

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-126.surge.sh

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-126.surge.sh

File: stories/LineChart.stories.js [hunk 1]
```diff
@@ -168,6 +168,26 @@ storiesOf('LineChart', module)
...
+        <DataProvider
+          defaultLoader={staticLoader}
+          baseDomain={staticBaseDomain}
+          series={[{ id: 1, color: 'steelblue' }, { id: 2, color: 'maroon' }]}
+        >
+          <LineChart />
+        </DataProvider>
+      </div>
+    </React.Fragment>
+  ))
```
Comment (valerii-cognite) [inline stories/LineChart.stories.js]: Isn't this story a duplicate of `Full-size` story?

Comment (evancharlton) [reply stories/LineChart.stories.js]: Yeah, more or less. Removed.

**Comment by evancharlton** [comment]
Huh, that's really interesting (shrinking the window really small. I'll dig into that and see if I can figure out what's going on.

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-126.surge.sh

**Comment by evancharlton** [comment]
This can't really be a thing 😞 SVGs need to have a width/height explicitly specified (and CSS doesn't count), or else it's up to the browser to size them however they want (Chrome defaults to 150px)

Instead, I'm going to go all-in on `react-sizeme` I think.

**Comment by valerii-cognite** [comment]
It seems it is actually possible by specifying weird `max-height: 100%;` for svg.

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-126.surge.sh

**Comment by evancharlton** [comment]
I don't think we can actually do this anyway -- items still need to know how big (in pixels) they are. We can drop `react-sizeme` if we want to reimplement the measurement with refs. I say we just keep on with `react-sizeme` until it causes problems.

**Pull request closed without merging**